cumulusci/tasks/salesforce/UpdateAdminProfile.py

```diff
@@ -32,10 +32,16 @@ class UpdateAdminProfile(Deploy):
 
     namespaces = {"sf": "http://soap.sforce.com/2006/04/metadata"}
 
+    def __init__(self, *args, **kwargs):
+        self._default_package_xml = False
+        self._package_xml_content = None
+        super(UpdateAdminProfile, self).__init__(*args, **kwargs)
```

Comment:
Since the default values are immutable you can just put

    _default_package_xml = False
    _package_xml_content = None

as class members.

Reply:
Fixed using the suggestion below

Reply:
So now there's no reason to define `__init__` here.